src/internal/testutilpachctl/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+load("@rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "testutilpachctl",
```

Comment:
You could make this testonly=1 so we don't have this happen again.  That will prevent a non-testonly=1 target from consuming it.